frankenphp.go

```diff
@@ -41,6 +41,7 @@ var (
 	InvalidRequestError         = errors.New("not a FrankenPHP request")
 	AlreaydStartedError         = errors.New("FrankenPHP is already started")
 	InvalidPHPVersionError      = errors.New("FrankenPHP is only compatible with PHP 8.2+")
+	ZendSignalsError            = errors.New("Zend Signals are enabled, recompible PHP with --disable-zend-signals")
```

Comment:
Typo here, `recompible` -> `recompile`?

Reply:
Thanks fixed in https://github.com/dunglas/frankenphp/pull/67